core/src/main/java/team/unnamed/gui/core/GUIListeners.java

```diff
@@ -82,17 +101,15 @@ public void onClick(InventoryClickEvent event) {
 
             boolean cancelClick = guiData.isCancelledClick();
 
-            if (!itemClickableOptional.isPresent()) {
+            if (itemClickableOptional.isEmpty()) {
```

Comment:
[`Optional#isEmpty()`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Optional.html#isEmpty()) was added in Java 11, the base version is 8